tests/jquery.flot.symbol.Test.js

```diff
@@ -0,0 +1,45 @@
+/* global $, describe, it, xit, xdescribe, after, afterEach, expect*/
+
+describe("flot symbol plugin", function() {
+    var placeholder, plot;
+    var options = {
+        series: {
+            shadowSize: 0, // don't draw shadows
+            lines: { show: false},
+            points: { show: true, fill: false, symbol: 'circle' }
+        }
+    };
+
+    beforeEach(function() {
+        placeholder = setFixtures('<div id="test-container" style="width: 600px;height: 400px">')
+            .find('#test-container');
+    });
+
+    it ('provides a list of draw symbols functions', function () {
+        plot = $.plot(placeholder, [[]], options);
+        expect(typeof plot.drawSymbol).toBe('object');
+    })
+
+    var shapes = ['ellipse', 'square', 'rectangle', 'cross', 'diamond', 'triangle', 'plus'];
```

Comment:
I wish the order of this list to match the order of the handlers from jquery.flot.symbol.js and from the example html file.